src/Catalyst.Core.Modules.Consensus/Deltas/DeltaBuilder.cs

```diff
@@ -55,6 +55,7 @@ public sealed class DeltaBuilder : IDeltaBuilder
         private readonly IDeltaTransactionRetriever _transactionRetriever;
         private readonly IDeterministicRandomFactory _randomFactory;
         private readonly IHashProvider _hashProvider;
+        private readonly IMapperProvider _mapperProvider;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_mapperProvider' or refactor the code to use its value.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)